generators/CoreCLRStatefulService/templates/main/deploy/deploy.ps1

```diff
@@ -1,4 +1,21 @@
+function Uninstall() {
+    Write-Host "Uninstalling App as installation failed... Please try installation again."
+    Invoke-Expression "& $PSScriptRoot\uninstall.ps1"
+    Exit
```

Comment:
should there be an exit code aswell ?

Reply:
Not necessary as the default is 0, and if there is any issue in uninstall script then it will exit before with non-zero errcode anyways.